### PR TITLE
LPD-15852 Create Data set Actions page object model

### PR DESCRIPTION
**Pull request by chestofwonders**
Task: https://liferay.atlassian.net/browse/LPD-15852

The goal of the PR is to provide a page object model representing the Actions tab for the DSM. That allows to navigate to this tab, having both a Data Set and a View previously created.

There is also a test for Creation Actions creation.

For the moment, the responsibility of creating previous required data (a Data Set with a View) belongs to the test itself, done by UI, since there's no consensus yet within the team about how and where that thing should be placed.

### CODE REVIEW

File: modules/test/playwright/tests/frontend-data-set-views-web/actions.spec.ts [hunk 1]
```diff
@@ -0,0 +1,47 @@
+/**
```
Comment (juanjofgliferay) [inline modules/test/playwright/tests/frontend-data-set-views-web/actions.spec.ts:1]: - Do we have any specific structure for the `*.spec.ts` files in mind or just drop them in the root folder?

File: modules/test/playwright/tests/frontend-data-set-views-web/utils/types.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/**
```
Comment (juanjofgliferay) [inline modules/test/playwright/tests/frontend-data-set-views-web/utils/types.ts:1]: - We usually have a `types` folder for typing related things. We could start with a shared `index.d.ts`

**Comment by juanjofgliferay** [comment]
@chestofwonders Great job. Couple of comments about file structure, other than that LGTM :rocket:

File: modules/test/playwright/tests/frontend-data-set-views-web/actions.spec.ts (hunk 1 above)
Comment (chestofwonders) [reply modules/test/playwright/tests/frontend-data-set-views-web/actions.spec.ts:1]: Nothing in mind...another thing to think about when files number becomes bigger

**Review by juanjofgliferay — APPROVED** [review]
:+1:

**Comment by chestofwonders** [comment]
ci:test:relevant

**Comment by chestofwonders** [comment]
ci:forward

**Pull request closed without merging**